site/getting-started/overview.md

```diff
@@ -18,79 +18,80 @@ head:
+Onboarding to web3 should be as simple as signing up for any website.
 
-Each of these steps adds friction. As a result, most new users drop off before they ever reach the magic moment in your app! We need to make wallets, seed phrases, and gas costs disappear to onboard the next billion users.
+Traditional wallets make your users jump through hoops like downloading an extension, backing up a seed phrase, and funding a wallet with ETH. Each of these steps adds friction and cause new users to drop off before they ever reach the magic moment in your app!
 
-Account Kit makes it easy to onboard users with smart accounts -- account abstraction wallets embedded directly in your app with seamless UX. Account Kit unlocks:
+*Embedded Accounts* make wallets invisible. Sign up for a non-custodial web3 account with no downloads, transact without gas, and recover without a seed phrase. Bring the next billion users onchain with embedded accounts.
 
-**Your app, your UX:** Design every step of the user experience natively in your app, from signup to checkout.
+- **Your app, your UX:** Design every step natively in your app, from sign up to checkout.
+- **Familiar auth methods:** Sign up with email, social login, passkeys (i.e. biometrics), or self-custodial signers like Metamask.
+- **Sponsor gas:** increase engagement by [sponsoring gas](/using-smart-accounts/sponsoring-gas/gas-manager) fees through an ERC-4337 paymaster.
+- **Batch transactions:** streamline multi-step actions like approve+swap in a single transaction by [batching](/using-smart-accounts/batch-user-operations) with the Bundler API.
+- **Account recovery:** Never lose your keys again. Embedded accounts provide more recovery methods than any other wallet including private key export, email recovery, and onchain recovery plugins.
 
-- **Familiar, secure login:** Streamline your sign-up flow with custom [authentication methods](/signers/choosing-a-signer), including email, social login, SMS, biometrics, passkeys, or self-custodial signers like Metamask.
-- **Sponsor gas:** enable users to transact without gas by [sponsoring gas](/using-smart-accounts/sponsoring-gas/gas-manager) fees through an ERC-4337 paymaster.
-- **Batch transactions:** streamline multi-step flows into a single transaction by [batching actions](/using-smart-accounts/batch-user-operations) in a UserOp to the Bundler API.
-- **Delightful developer experience:** Account Kit is built on Viem and fully compatible with ethers.js and the EIP-1193 interface to make integration a breeze.
+# How it works
 
-# What is Account Kit?
+**Embedded Accounts** use [ERC-4337 Account Abstraction](https://www.youtube.com/watch?v=Vpk_MhY-EeE) to build next-gen wallet UX. Account Abstraction dramatically simplifies the user experience by adding a few new developer components like a smart account, bundler, and paymaster. You can also configure your embedded accounts as a lightweight EOA if you prefer.
```

Comment:
for the last sentence, maybe we link out the dev docs: https://accountkit.alchemy.com/packages/aa-alchemy/signer/overview.html#using-the-signer-as-an-eoa

Reply:
ok added